src/Tasks/Microsoft.NET.Build.Tasks/Resources/Strings.resx

```diff
@@ -225,4 +225,16 @@
   <data name="NU1012" xml:space="preserve">
     <value>Dependency conflict. '{0}' expected '{1}' but received '{2}'</value>
   </data>
+  <data name="DuplicateItemsDefaultExplanation" xml:space="preserve">
+    <value> The .NET SDK includes {0} items from your project directory by default.</value>
+  </data>
+  <data name="DuplicateItemsError" xml:space="preserve">
+    <value>Duplicate {0} items were included.{1}  The duplicate items were: {2}</value>
+  </data>
+  <data name="DuplicateItemsHowToFix" xml:space="preserve">
+    <value> You can either remove these items from your project file, or set the '{0}' property to '{1}' if you want to explicitly include them in your project file.{2}</value>
```

Comment:
This is really hard to follow for translators: leading whitespace, placeholder followed by two spaces followed by another sentence. Placeholder immediately after a sentence. Can we just log consecutive independent messages? If not, maybe don't worry about DRY and just put the full text of each message a user can see so the translator has all of the context. At a minimum, use the `<comment>` element to explain how these compose.